terraform_modules/canary_template/providers.tf

```diff
@@ -0,0 +1,35 @@
+terraform {
+  required_providers {
+    aws   = {
+      source  = "hashicorp/aws"
+      version = ">= 3.0"
+    }
+  }
+
+  backend s3 {
+    bucket         = "nr-coreint-canaries-tfstates"
+    dynamodb_table = "nr-coreint-canaries-tflocking"
+    key            = "base-framework/integrations/oracle.tfstate"
```

Comment:
I would remove the key so there is no backend key set by default and there is no possibility to overwrite a previous deployment.

Take canaries as an example:
 * https://github.com/newrelic/coreint-canaries/blob/9787fa5/canaries/apache/terraform/providers.tf#L9-L13
 * https://github.com/newrelic/coreint-canaries/blob/9787fa5/.github/workflows/configureCanaryEnvironment.yml#L52

Reply:
Cool. Thanks for pointing the example